src/main/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/transformation/services/DocumentMapper.java

```diff
@@ -14,9 +14,11 @@
      */
     public Item<ScannedDocument> toCaseDoc(
         InputScannedDoc exceptionRecordDoc,
-        String exceptionRecordReference
+        String exceptionRecordReference,
+        boolean isAutomatedProcess
     ) {
-        if (exceptionRecordDoc == null) {
+        if (!isAutomatedProcess && exceptionRecordDoc == null) {
+            // When request is NOT auto case creation and exception record reference is null
```

Comment:
should it not throw exception for this scenario ?

Reply:
Yeah, it should. But it was implemented like this before. I'll update `ExceptionRecordValidator.java` to assert that exception record id exists. is that ok?

Reply:
I misread the parameter name `exceptionRecordDoc` as `exceptionRecordId`. This change is not required here, I'll revert